packages/manager/core/api/src/client.ts

```diff
@@ -42,25 +43,34 @@ export const v6 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/apiv6',
 });
-v6.interceptors.response.use(null, handleAuthenticationError);
 
 export const aapi = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/2api',
 });
-aapi.interceptors.response.use(null, handleAuthenticationError);
 
 export const ws = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/ws',
 });
-ws.interceptors.response.use(null, handleAuthenticationError);
 
 export const v2 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/2.0',
 });
-v2.interceptors.response.use(null, handleAuthenticationError);
 
-export const apiClient = { v6, aapi, ws, v2 };
+export const apiClient: any = { v6, aapi, ws, v2 };
+
+Object.keys(apiClient).map((api) => {
+  apiClient[api]?.interceptors?.request.use(function(config: any) {
+    const headers = getHeaders(config?.baseURL);
+    Object.keys(headers).map((header) => {
+      config.headers.set(header, headers[header]);
+      return header;
+    });
+    return config;
+  }, handleAuthenticationError);
```

Comment:
`handleAuthenticationError` is a `response` interceptor.

Reply:
I didn't change the behavior for this, what's the wrong ?

Reply:
Ok I see my mistake, I will update it